src/client/lobby/room-list.js

```diff
@@ -22,6 +23,16 @@ export default function RoomList(props) {
     );
 }
 
+function getClass(room) {
+    if(room.data.started){
+        return 'list-group-item list-group-item-danger space';
+    }
+    else{
+        return 'list-group-item list-group-item-warning space';
+    }
+}
+
 function joinRoom(room) {
+    setRoom(JSON.stringify(room));
```

Comment:
This doesn't really read very well. I had to scroll up to find the function. The storage-service export is good, but I'd make it the default export and then just have the service referenced as StorageService. That way this can be replaced with `StorageService.setRoom(...)`